starlette_discord/client.py

```diff
@@ -174,21 +189,25 @@ def redirect(self, state=None, prompt=None):
             url += f'&prompt={prompt}'
         return RedirectResponse(url)
 
-    def session(self, code) -> DiscordOAuthSession:
+    def session(self, code: str = None,
```

Comment:
same with above. I feel like `token` and `code` should be keyword-only arguments. or maybe a new method for initializing from a token. something like `session_from_token()`.

Reply:
I answer both comments here. 
I left it like that for compatibility, but i think it's a good idea to implement a special method to construct the session from a token. 

Just to clarify, i think this would only be done in the Client's class as the Session's would be functional in any case.

I have a couple hours, ill see if i can crunch this.